config/environments/production.rb

```diff
@@ -10,6 +10,9 @@
   # Base directory where user contributed annotations will be stored
   config.user_annotations_location = "/data/contributed_annotations/"
 
+  # Base directory where api created collections will be stored
+  config.api_collections_location = "/data/collections/"
+
```

Comment:
I should have mentioned when we talked about this, but there is a potential problem:
Now, for all instances of hcsvlap deployed in the world you will need to add this line of config to it's config file, and it MUST be the same value as was hardcoded in the controller otherwise you will probably break existing functionality.
So: You'll need to document this somewhere and be aware when upgrading.